src/tlo/util.py

```diff
@@ -498,15 +500,15 @@ def clean_dataframe(dataframes_dict: dict[str, DataFrame]) -> None:
         for _key, dataframe in dataframes_dict.items():
             all_data[_key] = dataframe.drop(dataframe.filter(like='Unnamed'), axis=1)  # filter and drop Unnamed columns
 
-    if files is None:
+    if files == 0 or files is None:
```

Comment:
Is the reasoning for allowing both `files = 0` and `files = None` to be used to select all files in directory so that different output behaviour can be selected? At the moment it looks like `files = 0` if there is a single file in directory will return just a single dataframe (but a dictionary otherwise), while `files = None` will always return a dict. Having this flexibility possibly makes sense, but if this is the intention I think it needs to be made more obvious by explaining in docstring. Alternatively might make more sense to instead have an explicit boolean flag `return_single_file_as_dataframe` to control output behaviour which if `True` and `len(all_data) == 1` will force a single dataframe to be returned, with a dict returned otherwise, as the name would make it explicit to the user what the behaviour is.

Reply:
@matt-graham , declaring an explicit flag `return_single_file_as_dataframe`  to control output is good but I'm afraid it will change the behaviour that modellers are used to in pd.read_excel. If we are to implement this now it will mean introducing this flag in several other files as well. Do we want to this in this PR?

Reply:
Hi @mnjowe, thanks for the explanation and for expanding the docstring. I think the behaviour with different values of `files` are now clearly defined to marking this as resolved